.version

```diff
@@ -1 +1 @@
-7.3.3
\ No newline at end of file
+8.10.0-beta.10
```

Comment:
We need to revert this before merging.

Reply:
I wonder, are we releasing this as `8.10.0`?

Reply:
We can manually bump the version after. I was worried that the script that updates the version might miss something if we have inconsistent versions. I think it's better if this PR is just the "feature" change without version changes, etc. and use the regular flow to bump it after.